tt.go

```diff
@@ -0,0 +1,282 @@
+package tupleconv
+
+import (
+	"fmt"
+)
+
+// TypeName is the data type for type names.
+type TypeName string
+
+// Types are supported tarantool types.
+const (
+	TypeBoolean   TypeName = "boolean"
+	TypeString    TypeName = "string"
+	TypeInteger   TypeName = "integer"
+	TypeUnsigned  TypeName = "unsigned"
+	TypeDouble    TypeName = "double"
+	TypeNumber    TypeName = "number"
+	TypeDecimal   TypeName = "decimal"
+	TypeDatetime  TypeName = "datetime"
+	TypeUUID      TypeName = "uuid"
+	TypeArray     TypeName = "array"
+	TypeMap       TypeName = "map"
+	TypeVarbinary TypeName = "varbinary"
+	TypeScalar    TypeName = "scalar"
+	TypeAny       TypeName = "any"
+	TypeInterval  TypeName = "interval"
+)
```

Comment:
Why is it in the public api?

Reply:
Changed.

Reply:
So, if I ever want to create a slice of field formats (to avoid manual converters creation), how can I specify field types? It is required to have public type names OR specific type for field format. Or have I missed the day to create such a slice of formats?

Reply:
Hi!
I did the following:
- supported types and `TypeName` data type have been made public
- For `SpaceField` the field responsible for the type now has the `TypeName` type (instead of string); you can see usage example here https://github.com/tarantool/go-tupleconv/blob/bb5b51eac2f0fd7ba7f6906441a5d7fff92fb2c0/example_test.go#L88
- Added the function `GetConverterByType(factory, typ)`, which returns a converter by type name. It allows build an array of converters more flexible; an example is available here:
https://github.com/tarantool/go-tupleconv/blob/bb5b51eac2f0fd7ba7f6906441a5d7fff92fb2c0/example_test.go#L107